VulkanHppGenerator.cpp

```diff
@@ -4199,6 +4322,17 @@ int main( int argc, char **argv )
     assert(vkData.deleterTypes.find("") != vkData.deleterTypes.end());
     writeTypes(ofs, vkData, defaultValues);
 
+    // write all template functions for the structure pointer chain validation
```

Comment:
No. Due to the reversed linking between parent and child the information we need is not available here.